internal/mage/util/engine.go

```diff
@@ -58,6 +61,17 @@ insecure-entitlements = ["security.insecure"]
 {{ end -}}
 `
 
+// nvidiaSetupHelper provides the required steps to setup nvidia-container-toolkit:
+const nvidiaSetupHelper = `
+#!/bin/sh
+
+export distribution=$(. /etc/os-release;echo $ID$VERSION_ID)
+curl -fsSL https://nvidia.github.io/libnvidia-container/gpgkey | gpg --dearmor -o /usr/share/keyrings/nvidia-container-toolkit-keyring.gpg
+curl -s -L https://nvidia.github.io/libnvidia-container/experimental/$distribution/libnvidia-container.list | sed 's#deb https://#deb [signed-by=/usr/share/keyrings/nvidia-container-toolkit-keyring.gpg] https://#g' | tee /etc/apt/sources.list.d/nvidia-container-toolkit.list
+apt-get update
+apt-get install nvidia-container-toolkit -y
```

Comment:
Curious what the new image size is for the ubuntu image. Totally okay with the tradeoff here for the moment, just wondering what final number actually ends up being.